recOrder/plugin/workers/calibration_workers.py

```diff
@@ -1,3 +1,4 @@
+from argparse import MetavarTypeHelpFormatter
```

Comment:
Does this do anything?

Reply:
Oops that's an unintended behavior of my editor 😅